frontend/compositions/dashboardHeader/helperFunctions.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useCallback, useEffect } from "react"
```

Comment:
We have a `helpers/` directory now (at the frontend root level) if you wanted to move this file there!

Reply:
A small change I would suggest for maintainability is changing the import style here so it's not default importing React proper, it should be something more like `import * as React, { useState, useCallback, useEffect } from "react"`. Thos blog post explains the rationale behind that: https://epicreact.dev/importing-react-through-the-ages/

Reply:
Good to know - thank you for sharing the blog post as well for more context.